@TyeYeah it fails to build:
```
FAILED: librz/analysis/librz_analysis.so.0.2.0-git.p/rtti_msvc.c.o 
clang -Ilibrz/analysis/librz_analysis.so.0.2.0-git.p -I. -I.. -I../librz/include -I../librz/analysis/arch/gb -I../librz/analysis/arch/hexagon -I../shlr/spp -Isubprojects/sdb -I../subprojects/sdb -Isubprojects/sdb/src -I../subprojects/sdb/src -Isubprojects/zlib-1.2.11 -I../subprojects/zlib-1.2.11 -Ishlr -I../shlr -I../librz/asm/arch/include -I../librz/asm/arch -I../librz/asm/arch/h8300 -I../librz/asm/arch/hexagon -I../librz/asm/arch/msp430 -I../librz/asm/arch/rsp -I../librz/asm/arch/mcore -I../librz/asm/arch/v850 -I../librz/asm/arch/propeller -I../librz/asm/arch/ebc -I../librz/asm/arch/cr16 -I../librz/asm/arch/8051 -I../librz/asm/arch/v810 -I../librz/asm/arch/or1k -I../librz/bin/mangling -I../librz/bin/format -I../subprojects/capstone-bundled/include -I../subprojects/capstone-bundled/include/capstone -Xclang -fcolor-diagnostics -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -O2 -g --std=gnu99 -fvisibility=hidden -D_GNU_SOURCE -Werror=sizeof-pointer-memaccess -fPIC -pthread -DRZ_PLUGIN_INCORE=1 -MD -MQ librz/analysis/librz_analysis.so.0.2.0-git.p/rtti_msvc.c.o -MF librz/analysis/librz_analysis.so.0.2.0-git.p/rtti_msvc.c.o.d -o librz/analysis/librz_analysis.so.0.2.0-git.p/rtti_msvc.c.o -c ../librz/analysis/rtti_msvc.c
../librz/analysis/rtti_msvc.c:833:20: error: expected ';' after expression
        vtable.size = size
                          ^
                          ;
1 error generated.
```
Please also rebase your code on top of the latest `dev` branch.

--- a/librz/analysis/rtti_itanium.c
+++ b/librz/analysis/rtti_itanium.c
@@ -762,7 +762,14 @@ static void recovery_apply_vtable(RVTableContext *context, const char *class_nam
 		return;
 	}
 
-	RzAnalysisVTable vtable = { .id = NULL, .offset = 0, .size = 0, .addr = vtable_info->saddr };
+	ut64 startAddress = vtable_info->saddr;
+	RVTableInfo *vtablet = rz_analysis_vtable_parse_at(context, startAddress);
+	ut64 size = 0;
+	if (vtablet) {
+		size = rz_analysis_vtable_info_get_size(context, vtablet);
+	}
+
+	RzAnalysisVTable vtable = { .id = NULL, .offset = 0, .size = size, .addr = vtable_info->saddr };
 	rz_analysis_class_vtable_set(context->analysis, class_name, &vtable);
 	rz_analysis_class_vtable_fini(&vtable);
 

--- a/librz/analysis/rtti_itanium.c
+++ b/librz/analysis/rtti_itanium.c
@@ -762,12 +762,8 @@ static void recovery_apply_vtable(RVTableContext *context, const char *class_nam
 		return;
 	}
 
-	ut64 startAddress = vtable_info->saddr;
-	RVTableInfo *vtablet = rz_analysis_vtable_parse_at(context, startAddress);
 	ut64 size = 0;
-	if (vtablet) {
-		size = rz_analysis_vtable_info_get_size(context, vtablet);
-	}
+	size = rz_analysis_vtable_info_get_size(context, vtable_info);
 
 	RzAnalysisVTable vtable = { .id = NULL, .offset = 0, .size = size, .addr = vtable_info->saddr };
 	rz_analysis_class_vtable_set(context->analysis, class_name, &vtable);

--- a/librz/analysis/rtti_msvc.c
+++ b/librz/analysis/rtti_msvc.c
@@ -814,38 +814,37 @@ static char *unique_class_name(RzAnalysis *analysis, const char *original_name) 
 	return name;
 }
 
-static void recovery_apply_vtable(RzAnalysis *analysis, const char *class_name, RVTableInfo *vtable_info) {
+static void recovery_apply_vtable(RVTableContext *context, const char *class_name, RVTableInfo *vtable_info) {
 	if (!vtable_info) {
 		return;
 	}
 
-	RVTableContext context;
-	rz_analysis_vtable_begin(analysis, &context);
+	rz_analysis_vtable_begin(context->analysis, context);
 	ut64 size = 0;
-	size = rz_analysis_vtable_info_get_size(&context, vtable_info);
+	size = rz_analysis_vtable_info_get_size(context, vtable_info);
 
 	RzAnalysisVTable vtable;
 	vtable.size = size;
 	vtable.id = NULL;
 	vtable.offset = 0;
 	vtable.addr = vtable_info->saddr;
-	rz_analysis_class_vtable_set(analysis, class_name, &vtable);
+	rz_analysis_class_vtable_set(context->analysis, class_name, &vtable);
 	rz_analysis_class_vtable_fini(&vtable);
 
 	RVTableMethodInfo *vmeth;
 	rz_vector_foreach(&vtable_info->methods, vmeth) {
 		RzAnalysisMethod meth;
-		if (!rz_analysis_class_method_exists_by_addr(analysis, class_name, vmeth->addr)) {
+		if (!rz_analysis_class_method_exists_by_addr(context->analysis, class_name, vmeth->addr)) {
 			meth.addr = vmeth->addr;
 			meth.vtable_offset = vmeth->vtable_offset;
-			RzAnalysisFunction *fcn = rz_analysis_get_function_at(analysis, vmeth->addr);
+			RzAnalysisFunction *fcn = rz_analysis_get_function_at(context->analysis, vmeth->addr);
 			meth.name = fcn ? rz_str_new(fcn->name) : rz_str_newf("virtual_%" PFMT64d, meth.vtable_offset);
 			//Temporarily set as attr name
 			meth.real_name = fcn ? rz_str_new(fcn->name) : rz_str_newf("virtual_%" PFMT64d, meth.vtable_offset);
 			meth.method_type = RZ_ANALYSIS_CLASS_METHOD_VIRTUAL;
 		} else {
 			RzAnalysisMethod exist_meth;
-			if (rz_analysis_class_method_get_by_addr(analysis, class_name, vmeth->addr, &exist_meth) == RZ_ANALYSIS_CLASS_ERR_SUCCESS) {
+			if (rz_analysis_class_method_get_by_addr(context->analysis, class_name, vmeth->addr, &exist_meth) == RZ_ANALYSIS_CLASS_ERR_SUCCESS) {
 				meth.addr = vmeth->addr;
 				meth.name = rz_str_new(exist_meth.name);
 				meth.real_name = rz_str_new(exist_meth.real_name);
@@ -854,7 +853,7 @@ static void recovery_apply_vtable(RzAnalysis *analysis, const char *class_name, 
 				rz_analysis_class_method_fini(&exist_meth);
 			}
 		}
-		rz_analysis_class_method_set(analysis, class_name, &meth);
+		rz_analysis_class_method_set(context->analysis, class_name, &meth);
 		rz_analysis_class_method_fini(&meth);
 	}
 }
@@ -937,7 +936,7 @@ static const char *recovery_apply_complete_object_locator(RRTTIMSVCAnalContext *
 	rz_analysis_class_create(analysis, name);
 	ht_up_insert(context->col_td_classes, col->addr, name);
 
-	recovery_apply_vtable(analysis, name, col->vtable);
+	recovery_apply_vtable(context->vt_context, name, col->vtable);
 	recovery_apply_bases(context, name, &col->base_td);
 
 	return name;
@@ -973,7 +972,7 @@ static const char *recovery_apply_type_descriptor(RRTTIMSVCAnalContext *context,
 		return name;
 	}
 
-	recovery_apply_vtable(analysis, name, td->col->vtable);
+	recovery_apply_vtable(context->vt_context, name, td->col->vtable);
 	recovery_apply_bases(context, name, &td->col->base_td);
 
 	return name;

--- a/librz/analysis/rtti_msvc.c
+++ b/librz/analysis/rtti_msvc.c
@@ -821,12 +821,8 @@ static void recovery_apply_vtable(RzAnalysis *analysis, const char *class_name, 
 
 	RVTableContext context;
 	rz_analysis_vtable_begin(analysis, &context);
-	ut64 startAddress = vtable_info->saddr;
-	RVTableInfo *vtablet = rz_analysis_vtable_parse_at(&context, startAddress);
 	ut64 size = 0;
-	if (vtablet) {
-		size = rz_analysis_vtable_info_get_size(&context, vtablet);
-	}
+	size = rz_analysis_vtable_info_get_size(&context, vtable_info);
 
 	RzAnalysisVTable vtable;
 	vtable.size = size;

--- a/librz/analysis/rtti_msvc.c
+++ b/librz/analysis/rtti_msvc.c
@@ -819,7 +819,17 @@ static void recovery_apply_vtable(RzAnalysis *analysis, const char *class_name, 
 		return;
 	}
 
+	RVTableContext context;
+	rz_analysis_vtable_begin(analysis, &context);
+	ut64 startAddress = vtable_info->saddr;
+	RVTableInfo *vtablet = rz_analysis_vtable_parse_at(&context, startAddress);
+	ut64 size = 0;
+	if (vtablet) {
+		size = rz_analysis_vtable_info_get_size(&context, vtablet);
+	}
+
 	RzAnalysisVTable vtable;
+	vtable.size = size;
 	vtable.id = NULL;
 	vtable.offset = 0;
 	vtable.addr = vtable_info->saddr;

--- a/test/db/cmd/cmd_ac
+++ b/test/db/cmd/cmd_ac
@@ -569,3 +569,41 @@ EXPECT=<<EOF
 [{"name":"InAbsentia","bases":[{"id":"0","name":"Album","offset":0}],"vtables":[{"id":"0","addr":4268452,"offset":0}],"methods":[{"name":"virtual_0","addr":4198536,"type":"VIRTUAL","vtable_offset":0},{"name":"virtual_4","addr":4198579,"type":"VIRTUAL","vtable_offset":4}]},{"name":"type_info","bases":[],"vtables":[{"id":"0","addr":4268540,"offset":0}],"methods":[{"name":"virtual_0","addr":4198710,"type":"VIRTUAL","vtable_offset":0}]},{"name":"Album","bases":[],"vtables":[{"id":"0","addr":4268388,"offset":0}],"methods":[{"name":"virtual_0","addr":4198493,"type":"VIRTUAL","vtable_offset":0},{"name":"virtual_4","addr":4198567,"type":"VIRTUAL","vtable_offset":4}]}]
 EOF
 RUN
+
+NAME=acvf2
+FILE=bins/elf/analysis/elf-virtualtable
+CMDS=<<EOF
+aaa
+acvf 0
+acvf 0x8 A
+acvf 0x0 C
+acvf 0x10 ABC
+EOF
+EXPECT=<<EOF
+Function address: 0x00400ac8, in A vtable 0
+Function address: 0x00400ac8, in B vtable 0
+Function address: 0x00400ac8, in C vtable 0
+Function address: 0x00400af4, in A vtable 0
+Function address: 0x00400ac8, in C vtable 0
+EOF
+RUN
+
+NAME=acvf3
+FILE=bins/mach0/TestRTTI-arm64
+CMDS=<<EOF
+aaa
+acvf 0 A
+acvf 0x8 B
+acvf 0x10 C
+acvf 0x18 ABC
+EOF
+EXPECT=<<EOF
+Function address: 0x100005f1c, in A vtable 0
+Function address: 0x100005fe0, in B vtable 0
+Function address: 0x100005f44, in B vtable 1
+Function address: 0x100005fe0, in B vtable 2
+Function address: 0x100005f44, in B vtable 3
+Function address: 0x1000060a0, in C vtable 1
+Function address: 0x1000060a0, in C vtable 3
+EOF
+RUN

